scripts/warroom/treasury_divest/convex.py

```diff
@@ -0,0 +1,28 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_convex()
+    vault.init_curve_v2()
+
+    badger_fraxbp = vault.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+    wcvx_badger_fraxbp = vault.contract(r.convex.frax.wcvx_badger_fraxbp)
+    badger = vault.contract(r.treasury_tokens.BADGER)
+    usdc = vault.contract(r.treasury_tokens.USDC)
+    frax = vault.contract(r.treasury_tokens.FRAX)
+    fxs = vault.contract(r.treasury_tokens.FXS)
+
+    vault.take_snapshot(tokens=[badger, usdc, frax, fxs])
+
+    # https://etherscan.io/tx/0x74f4492c78385b633fdecb2200efdd99f7377dbf2e0b94468c06682fb78961c1#eventlog (event #214)
+    badger_frax_kek_id = (
```

Comment:
no programmatic way of getting the kek id?

Reply:
ah, wasnt aware, thanks. added in: 0762208

Reply:
just one questions here; could we derive the pool id programmatically?

Reply:
yup b010597